indices/split_pairwise_networks.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import glob
+import os
+import pickle as pkl
+import re
+from tqdm import tqdm
+
+import networkx as nx
+
+from utils import parse_mesh_headings
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--metadata_dir',
+                        help='File from download_article_metadata containing info on the '
+                             'articles to add to the network',
+                        default='data/pubmed/efetch')
+    parser.add_argument('--in_dir',
+                        help='The directory containing networks to split. Paired networks will '\
```

Comment:
I don't think you need the `\` character here - python should (?) automatically concatenate strings within parentheses:
```suggestion
                        help='The directory containing networks to split. Paired networks will '
```

Reply:
You are correct, it does (also I don't remember putting in that backslash which is somewhat spooky)